cypress/e2e/swap.test.ts

```diff
@@ -92,16 +92,19 @@ describe('Swap', () => {
         cy.get('#swap-button').click()
         cy.get('#confirm-swap-or-send').click()
         cy.get('[data-testid="dismiss-tx-confirmation"]').click()
-        // ui check
-        cy.get('#swap-currency-output [data-testid="balance-text"]').should(
-          'have.text',
-          `Balance: ${initialBalance + BALANCE_INCREMENT}`
-        )
-
-        // chain state check
-        cy.then(() => hardhat.utils.getBalance(hardhat.wallet.address, USDC_MAINNET))
-          .then((balance) => Number(balance.toFixed(1)))
-          .should('eq', initialBalance + BALANCE_INCREMENT)
+
+        cy.then(() => hardhat.send('hardhat_mine', ['0x1', '0xc'])).then(() => {
```

Comment:
`hardhat.send` is an async action, so it needs to be wrapped in `cy.then`

Reply:
I think this is fine as an unblocker, but should really be exposed through cypress-hardhat's Utils class.